### ISSUE TITLE
Design seprate page for Cart Items

### ISSUE THREAD

**Issue by Ayonijakaushik19** [Jagannath8/shopping#20]

**Comment by aditya687** [comment]
Sir, I want on work on this issue @Jagannath8

**Comment by vchsi** [comment]
Hi @Jagannath8,
Because I did not get assigned to the other issue, I decided that I wanted to give a shot at this issue so can I get assigned to it?

**Edit; Also, is the cart retrieved from the database?**

**Comment by Jagannath8** [comment]
Sure @vchsi 
Don't go for database, products are already there in js file

**Comment by vchsi** [comment]
Does each account have different cart items or do they all have the same cart items?

If they are different, I think a database will be needed to store all the cart items.

**Comment by Jagannath8** [comment]
For now keep same cart items

**Comment by Jagannath8** [comment]
Hey @vchsi What is the status?

**Comment by vchsi** [comment]
@Jagannath8 I am currently Implementing the JavaScript functionality. I finished designing the page.

**Comment by Jagannath8** [comment]
Okay @vchsi 
Do it soon. I guess 7Aug is the last day for merging PRs

**Comment by vchsi** [comment]
@Jagannath8 Here is what I have for the cart page so far. Should it have more style or is this good?
![curcartpg](https://user-images.githubusercontent.com/8525130/128554266-3db822b9-b6d1-45b2-9bc0-04cc62274b65.PNG)

**Comment by vchsi** [comment]
With placeholder product and price

**Comment by Jagannath8** [comment]
@vchsi Looks good, just add total cart items
Ex - when products from other category are purchased cart items count should also increase

**Comment by vchsi** [comment]
@Jagannath8 In which file is the current cart items stored?

**Comment by Jagannath8** [comment]
@vchsi In product.html

**Comment by vchsi** [comment]
@Jagannath8 No, I mean if the products in cart don't save on reload or page change, how would they be transferred over to the cart page . I think they should be stored in a cookie as each person can have their own cart and the cart items will save even after reload.

**Comment by vchsi** [comment]
Submitted PR
